ILRepack/RepackOptions.cs

```diff
@@ -79,8 +78,10 @@ public List<string> AllowedDuplicateNameSpaces
         private readonly RepackAssemblyResolver globalAssemblyResolver = new RepackAssemblyResolver();
         private List<Regex> excludeInternalizeMatches;
 
-        public void SetSearchDirectories(string[] dirs)
+        public void SetSearchDirectories(IEnumerable<string> dirs)
         {
+            if (TargetPlatformDirectory != null)
+                dirs = new[] { TargetPlatformDirectory }.Concat(dirs);
```

Comment:
Which reminds me, do we want to support mono WPF patching? Though it would be... kinda... impossible, since we don't have WindowsBase, PresentationCore and PresentationFramework on mono :(